src/App/App.jsx

```diff
@@ -1,5 +1,5 @@
 import { Route, useLocation, Routes, Navigate, useNavigate } from 'react-router-dom'
-import { ThemeProvider } from 'styled-components/macro'
```

Comment:
styled components macro doesnt compile with vite, so we wont be able to have the SC names in dev tools until we find something else. (would it create too much inconsistency to have new stuff use CSS modules?)

@JBurkinshaw @parmvirthind @julia-rom @gwan93 @alanjleonard